Gemfile

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
-source 'https://rubygems.org'
+source "https://rubygems.org"
```

Comment:
Shouldn't the rule be single quotes? From what I can tell is that your ignoring the rubocop rule to accomodate CodeClimate rather than making CodeClimate follow our standard setup?

Or am I confused?